internal/i18n/en_translations_descriptions.go

```diff
@@ -0,0 +1,544 @@
+	TokenPoolConnector  = ffm("TokenPool.connector", "The token connector that is responsible for the token pool. Required on input when multiple token connectors are configured")
+	TokenPoolMessage    = ffm("TokenPool.message", "The UUID of the broadcast message used to inform the network to index this pool")
+	TokenPoolState      = ffm("TokenPool.state", "The current state of the token pool")
+	TokenPoolCreated    = ffm("TokenPool.created", "The creation time of the pool")
+	TokenPoolConfig     = ffm("TokenPool.config", "Input only field, with token connector specific configuration of the pool, such as an existing Ethereum address and block number to used to index the pool. See your chosen token connector documentation for details")
+	TokenPoolInfo       = ffm("TokenPool.info", "Token connector specific information about the pool. See your chosen token connector documentation for details")
+	TokenPoolTX         = ffm("TokenPool.tx", "Reference to the FireFly transaction used to create and broadcast this pool to the network")
+
+	// TokenTransfer field descriptions
+	TokenTransferType            = ffm("TokenTransfer.type", "The type of transfer such as mint/burn/transfer")
+	TokenTransferLocalID         = ffm("TokenTransfer.localId", "The UUID of this token transfer, in the local FireFly node")
+	TokenTransferPool            = ffm("TokenTransfer.pool", "The UUID the token pool this transfer applies to")
+	TokenTransferTokenIndex      = ffm("TokenTransfer.tokenIndex", "The index of the token within the pool that this transfer applies to")
+	TokenTransferURI             = ffm("TokenTransfer.uri", "The URI of the token this transfer applies to")
+	TokenTransferConnector       = ffm("TokenTransfer.connector", "The token connector. Required on input when there are more than one token connectors configured")
+	TokenTransferNamespace       = ffm("TokenTransfer.namespace", "The namespace for the transfer, which must match the namespace of the token pool")
+	TokenTransferKey             = ffm("TokenTransfer.key", "The blockchain signing key for the transfer. On input defaults to the first signing key of the organization that operates the node")
+	TokenTransferFrom            = ffm("TokenTransfer.from", "The source account for the transfer. On input defaults to the value of 'key'")
+	TokenTransferTo              = ffm("TokenTransfer.to", "The target account for the transfer. Required on input")
+	TokenTransferAmount          = ffm("TokenTransfer.amount", "The amount for the transfer. For non-fungible tokens will always be 1. For fungible tokens, the number of decimals for the token pool should be considered when inputting the amount. For example, with 18 decimals a fractional balance of 10.234 will be specified as 10,234,000,000,000,000,000")
+	TokenTransferProtocolID      = ffm("TokenTransfer.protocolId", "The protocol identifier of the blockchain event for the token transfer. See the 'protocolId' field on blockchain events for more information")
```

Comment:
This one does happen to somewhat align with the formatting used for blockchain events, although I don't know that we should enforce it. Suggestion: `A unique identifier for this transfer, as provided by the token connector`

For the record, erc1155 uses `BLOCK/TX/EVENT/N` where the final segment is optional and is used only when exploding batch events into individual transfer events.